routes/api/authRouter.js

```diff
@@ -108,4 +130,29 @@ router.get("/current", authMiddleware, async (req, res, next) => {
   }
 });
 
+router.patch("/avatars", authMiddleware, upload.single('avatar'), async (req, res, next) => {
+  try {
+    const userId = req.user._id;
+
+    if (!req.file) {
+      return res.status(400).json({ message: 'No file uploaded' });
+    }
+
+    const Jimp = require('jimp');
```

Comment:
To jest poprawne i możliwe w commonjs. Ale już w ecma script nie będzie. Rozważ czy nie utrzymywać wszystkich importów na samej górze dla większej czytelności. Bo za kilka tygodni możesz już nie pamiętać jakiego modułu się użyło i trzeba będzie szukać.